### ISSUE TITLE
LogWarnings/Errors: L3Absolute not found:  PATJetUpdater:updatedPatJetsBTAG

### ISSUE THREAD

**Issue by slava77** [cms-sw/cmssw#17573]
this needs to be understood and demoted or addressed otherwise.
The warning appears in standard 136.731 workflow in 90X now
it was seen since at least CMSSW_8_0_X_2016-09-07-1100
and CMSSW_8_1_X_2016-08-27-1100

@davidlange6 @gpetruc

**Comment by gpetruc** [comment]
I think we should get rid of the updatedPatJetsBTAG setup in normal 90X miniAOD production: it doesn't make sense now to first create corrected PAT jets with some b-tagging information, then uncorrect them, make some more b-tag, embed it and then re-correct them back (this was introduced in  #15421 maybe for compatibility with older AODs or miniAOD-from-miniAOD recipes).
In fact, probably we can just drop the alternative b-tag discriminators computed with the old tight track selection, as no problems have been found with the new track selection in the current data.

@ferencek @arizzi

**Comment by ferencek** [comment]
@gpetruc, I agree, changes in `PhysicsTools/PatAlgos/python/slimming/miniAOD_tools.py` and `PhysicsTools/PatAlgos/python/slimming/slimmedJets_cfi.py` introduced in #15421 should now be safe to remove.

@imarches @carolinecollard @mverzett @JyothsnaKomaragiri

**Comment by mverzett** [comment]
+1 for me

**Comment by ferencek** [comment]
Dear all, is anyone taking care of this request?

**Comment by slava77** [comment]
not that I'm aware of.
@ferencek If you can address this, may I suggest to follow up if no one confirms the work is in progress already in the next ~24 hours.

Thank you.

**Comment by ferencek** [comment]
@slava77, sounds good. Will do as you suggest.

**Comment by ferencek** [comment]
The issue has been addressed in the following PRs: https://github.com/cms-sw/cmssw/pull/17839 and https://github.com/cms-sw/cmssw/pull/17840
